serverless.yml

```diff
@@ -0,0 +1,23 @@
+org: Data-Epic
+app: sol-sight-serverless
+service: sol-sight-serverless
+
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: python3.9
+  iamRoleStatements:
+  - Effect: Allow
```

Comment:
can we attach to the function instead

Reply:
I will check it out

Reply:
Hi @hammedb197, I don't think I quite understand what you mean here. Can you please clarify